DevOps/frankfurter/resources/interfaces

```diff
@@ -7,22 +7,27 @@ iface lo inet loopback
 
 # The primary network interface
 allow-hotplug eth0
-iface eth0 inet static
-    address 192.168.13.100
-    network 192.168.13.0
-    netmask 255.255.255.0
-    gateway 192.168.13.1
-#iface eth0 inet dhcp
+iface eth0 inet dhcp
 # Example to keep MAC address between reboots
 #hwaddress ether DE:AD:BE:EF:CA:FE
 
 # The secondary network interface
 #auto eth1
 #iface eth1 inet dhcp
 
-# WiFi Example
 auto ra0
-iface ra0 inet dhcp
+mapping ra0
+    #script /home/ubuntu/whichwifi.sh
+    script guessnet-ifupdown
+    map student
+    map field
+    map default: field
+iface student inet dhcp
+#or switch to static
+    wpa-ssid "router_name"
```

Comment:
Ah i see the python file sets it
Please add a comment in here
`x, y, z are set in {scripy}.py`